### ISSUE TITLE
ae2 autocraft lag when the "structure" are connected to the network

### ISSUE THREAD

**Issue by nitsuga5124** [AppliedEnergistics/Applied-Energistics-2#893]
i have fps lag whent the molecular transformers are connected to the network, i try to put optifine end fastcraft an the lag comes up.

fix it

**Comment by TheJulianJES** [comment]
Does it also happend without FastCraft or OptiFine? Since they aren't supported in most mods.

**Comment by yueh** [comment]
You know that these issues are related?

**Comment by halvors** [comment]
The other issue is server lag spike related.

**Comment by yueh** [comment]
The other issue is mekanism causing issues. This is completely unclear, so for now not really related.

**Comment by TheJulianJES** [comment]
Okay, it was thinking with lag spike every time AE2 updates. Sorry.

**Comment by nitsuga5124** [comment]
when the autocraft are conectet, with and without optifine and fastcraft my fps go down aprox. 30 fps. the end

**Comment by Bestadd** [comment]
I can confirm this I happening in the FTB Infinity 1.0.2 pack; however, I could only recreate in single player.  Once I moved the world to a server the lag was gone.  Furthermore, I noticed that the FPS degreased further while I was in the chunks containing my six crafting CPU multi-block structures (1x2 with 1 1k CPU, 1 crafting monitor, 4 co-processors).  This FPS lag was present regardless of CPU utilization (crafting or now).  However, while crafting FPS dropped further.    Also, fastcraft was installed.  If you would find it helpful I can try to recreate again with out fastcraft.  I can also try recreating without the crafting monitors and or other parts to the multi-block, just let me know.
